muxink/src/codec.rs

```diff
@@ -22,8 +22,10 @@
 //! * [`length_delimited`]: Transforms byte-like values into self-contained frames with a
 //!   length-prefix.
 
-#[cfg(feature = "bincode")]
+#[cfg(feature = "muxink_bincode_codec")]
```

Comment:
The features can be renamed, I maintain they should be named `bincode` and `bytesrepr`, see above :)